style.css

```diff
@@ -46,12 +46,7 @@ body{
     position: absolute;
     width: 100%;
   }
-.body-div{
 
-}
-#video{
-    
-}
```

Comment:
It's blank provide some data

Reply:
- [ ] Good work using inline comments to provide feedback 👏🏾 . There are a few more things we need to do to add value to our review.

 - Your comment should be clear and specific on what should be done. For example **Please add a video link** is clear and more specific than **It's blank provide some data** 🤝 
 
 - You should always include an explanation, a "why", for each requested change. This helps the student grow professionally and know why certain things should be done a certain way. 😄 
 
 - Please always phrase the comments politely, as suggestions instead of orders. For example **Your video section is blank, please provide a video link**  sounds more polite than **provide some data** 🤣

Reply:
- [ ] Great job 👏 making your comments sound more friendly and polite but as requested by the previous reviewer please make sure to add an explanation as to "why" you requested a certain change. 
    - You can for example say "Please add a video link, as the current video isn't displaying this makes your project look unprofessional👶"
-  [ ] [OPTIONAL] You can also consider adding an emoji to your comments this makes it seem more playful & less serious.